server/start_server.bat

```diff
@@ -0,0 +1,4 @@
+@echo off
+
+set CSHARP_MODULE_DISABLE_COLLECTIBLE=true
```

Comment:
why?

Reply:
Sometimes some packages produce System.NotSupportedException, this is just to avoid that

Reply:
that packages are outdated then. It shouldn not be used.

Reply:
Alright, removed ;)